sway-lib-std/src/bytes.sw

```diff
@@ -913,3 +955,29 @@ fn test_eq() {
     other.swap(0, 1);
     assert(bytes != other);
 }
+
+#[test()]
+fn test_sha256() {
```

Comment:
Should we test each `type` or is this fine?

Reply:
Since we don't have the `From` trait implemented these would all need to be manually added.